app/controllers/api/v1/slack_controller.rb

```diff
@@ -2,7 +2,7 @@ class Api::V1::SlackController < Api::V1::BaseController
   def get_users
     result = slack.get_slack_users
     members = result.select do |member|
-      !member.deleted && !non_human_names.include?(member.profile.display_name_normalized)
+      (!member.deleted && !non_human_names.include?(member.profile.display_name_normalized) && !member.is_ultra_restricted)
```

Comment:
Line is too long. [123/100]

Reply:
Para apaciguar al mono acá puedes separarlo por `&&`:
```
!member.deleted &&
  !non_human_names.include?(member.profile.display_name_normalized) &&
  !member.is_ultra_restricted
```